### PR TITLE
Fix/tca 389/fix issue in item content backup during import

### PR DESCRIPTION
**Pull request by SashaTAO**
JIRA: https://oat-sa.atlassian.net/browse/TCA-389

Problem:
QTI APIP item import fails when there is item with the same identifier and we have to overwrite it. During import we try to backup all item's files and with S3 flysystem adapter it fails randomly when we move files to new directory. 

Solution:
Instead of moving files update item's http://www.tao.lu/Ontologies/TAOItem.rdf#ItemContent property to point to the new directory.

How to test:
- Import QTI item.
- Import the same item again using API `/taoQtiItem/RestQtiItem/importDeferred` with IBN guardian configured and following API request parameters:
`enableMetadataGuardians:true`
`enableMetadataValidators:true`
`itemMustExist:false`
`itemMustBeOverwritten:true`   

Requires: https://github.com/oat-sa/extension-tao-item/pull/399

### CODE REVIEW

File: test/unit/model/qti/ServiceTest.php [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+use core_kernel_classes_Resource;
+use oat\generis\model\fileReference\FileReferenceSerializer;
+use oat\generis\test\MockObject;
+use oat\oatbox\filesystem\Directory;
+use Psr\Log\LoggerInterface;
+use taoItems_models_classes_ItemsService;
+use oat\generis\test\TestCase;
+use oat\taoQtiItem\model\qti\Service;
+
+class ServiceTest extends TestCase
```
Comment (zagovorichev) [inline test/unit/model/qti/ServiceTest.php:36]: Item qti service test?

File: test/unit/model/qti/ServiceTest.php [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA ;
+ */
+
```
Comment (zagovorichev) [inline test/unit/model/qti/ServiceTest.php:19]: strict

File: test/unit/model/qti/ServiceTest.php (hunk 1 above)
Comment (SashaTAO) [reply test/unit/model/qti/ServiceTest.php]: Done.

File: test/unit/model/qti/ServiceTest.php (hunk 2 above)
Comment (SashaTAO) [reply test/unit/model/qti/ServiceTest.php]: Added.

**Comment by zagovorichev** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Pull request merged**